specification/eventgrid/Azure.Messaging.EventGrid.SystemEvents/Microsoft.AVS/AVS.tsp

```diff
@@ -59,38 +61,38 @@ model AvsScriptExecutionCancelledEventData
 #suppress "@azure-tools/typespec-azure-core/composition-over-inheritance" "Maintain compatibility with existing models."
 model AvsScriptExecutionFailedEventData extends AvsScriptExecutionEventData {
   /** Failure reason of an event. */
-  failureMessage?: string;
+  failureMessage: string;
 }
 
 /** Schema of the Data property of an EventGridEvent for Microsoft.AVS/privateClouds events. */
 model AvsPrivateCloudEventData {
   /** Id of the operation that caused this event. */
-  operationId?: string;
+  operationId: string;
 }
 
 /** Schema of the Data property of an EventGridEvent for Microsoft.AVS/clusters events. */
 model AvsClusterEventData {
   /** Id of the operation that caused this event. */
-  operationId?: string;
+  operationId: string;
 
   /** Hosts added to the cluster in this event, if any. */
-  addedHostNames?: string[];
+  addedHostNames: string[];
 
   /** Hosts removed from the cluster in this event, if any. */
-  removedHostNames?: string[];
+  removedHostNames: string[];
 
   /** Hosts in Maintenance mode in the cluster, if any. */
-  inMaintenanceHostNames?: string[];
+  inMaintenanceHostNames: string[];
```

Comment:
Do we know if the key is still included in the event if there are no hosts?

Reply:
I do not know